.github/workflows/pipeline.yml

```diff
@@ -1,18 +1,84 @@
-name: SchemaSpy on push
```

Comment:
Does this pipeline still produce the SchemaSpy output or is that going to be a separate workflow?